MediaPlayer/MediaPlayer.js

```diff
@@ -401,6 +413,14 @@ const MediaPlayerBehaviorDecorator = hoc((config, Wrapped) => { // eslint-disabl
 
 		onSliderChange = ({value}) => {
 			this.media.currentTime = value * this.state.duration;
+
+			const seconds = Math.floor(value * this.state.duration);
```

Comment:
Why didn't you use `this.media.currentTime` instead of `value * this.state.duration`?

Reply:
I don't want to read that variable since it's from media. I'd rather assign the value to a const variable.